File: module-4/m4s9-aop-exception/customer-management/customer-management/src/main/java/codegym/model/Customer.java
package codegym.model;

import lombok.Data;

import javax.persistence.*;

@Entity
public @Data class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    @Column(unique = true)
    private String email;
    private String address;
    @ManyToOne
    @JoinColumn(name = "province_id", referencedColumnName = "id")
    private Province province;
}

File: module-5/m5-casestudy/src/app/module/customer/customer-model/Customer.ts
import {ObjectLookUp} from '../../../model/ObjectLookUp';

export interface Customer {
  id?: number;
  customerType: ObjectLookUp;
  name: string;
  birthday: string;
  idCard: string;
  phone: string;
  email: string;
  address: string;
  isCheck?: boolean;
}
